sonarpedia.json

```diff
@@ -3,7 +3,7 @@
   "languages": [
     "JAVA"
   ],
-  "latest-update": "2023-12-18T09:12:31.787089Z",
+  "latest-update": "2024-02-13T17:35:17.884561921Z",
```

Comment:
Is it possible you updated the metadata at some point? I think this is only updated when running a metadata update, which isn't consistent with the changes in this PR.

Reply:
Yes, accidentally, instead of generating the metadata for S6891 only. Reverted it.